src/openforms/registrations/contrib/objects_api/tests/test_api_group_model.py

```diff
@@ -117,3 +118,44 @@ def test_validate_iot_urls_within_catalogue(self):
                     _config.clean()
 
                 self.assertEqual(list(exc_context.exception.error_dict.keys()), [field])
+
+    def test_validate_iot_descriptions_within_catalogue(self):
+        iot_in_test = "PDF Informatieobjecttype"
+        iot_in_other = "Attachment Informatieobjecttype other catalog"
+        config = ObjectsAPIGroupConfigFactory.create(
+            for_test_docker_compose=True,
+            catalogue_domain="TEST",
+            catalogue_rsin="000000000",
+        )
+
+        for field in (
```

Comment:
These are defined in `validators.py` as `_IOT_DESCRIPTION_FIELDS`, so I guess those could be reused

Reply:
I don't really agree:

* `_IOT_DESCRIPTION_FIELDS` is a private implementation detail, it should not be imported anywhere else
* I want the tests to be explicit and manually writing out the fields in the test can catch typo's
* DRY in tests leads to indirection, which leads to making it harder to figure out what is actually being tested